packages/orbit-components/src/AirportIllustration/AirportIllustration.ct-story.tsx

```diff
@@ -9,7 +9,8 @@ export default function AirportIllustrationStory() {
   return (
     <div className="space-y-sm flex flex-col">
       {NAMES.map(illustration => (
-        <div className="bg-white-normal min-h-[80px] w-full">
+        /* eslint-disable-next-line tailwindcss/no-custom-classname */
```

Comment:
Hello, I had to add these disabling comments, because the ESLint is shouting that these `min-h...`, `min-w-..`,  `max-h...`, `max-w-..` classes are not TailwindCSS classes (but that's not true).
I guess it's not implemented in the Eslint Tailwind plugin 🤷‍♀️ .


**I found these "solutions":**
- add disabling rule comments everytime it's needed
- add this rule to the whitelist for `tailwindcss/no-custom-classname` in `eslintrc.cjs` file

WDYT? Or do you suggest any other solution? @DSil

Reply:
> classes are not TailwindCSS classes (but that's not true)

It is true. We want to only accept the sizes defined in the [tailwind config](https://github.com/kiwicom/orbit/blob/master/packages/orbit-tailwind-preset/src/index.ts), and the linter is prepared for that. We should **not** rely on the default tailwind sizes, at all. I am sorry we did not introduce you to that, but you will have to revert those, unfortunately. Custom sizes are meant to be kept as custom sizes (or have tokens generated for them).

Reply:
Ok, my theory was based on the default Tailwind sizes as it's defined in the official doc. In that case, I'll revert it. 😢

Reply:
We're actually missing the size classes in [our documentation](https://orbit.kiwi/foundation/tailwind/classnames/).
I didn't check, but maybe they're the same as spacing 🤔 — if they are, it should be clear. If they're not, they should be documented.